### DISCUSSION TITLE
Saved data?

### DISCUSSION THREAD

**Discussion by KyllianGamer** [mircokroon/minecraft-world-downloader#261] [Q&A] [answered]
So I downloaded a world but now there's like 700 .dat files in /world/data/ are these like individual chunks? I'm afraid that that's what it is.. And if it is how would I load these .dat files into like an actual world?

**Comment by KyllianGamer** [comment, marked as answer]
Never mind, I'm dumb.
For anyone else that doesn't know how it works:
You just need to create a new singleplayer world and go in the save files and replace the /data, /region ... folder with the folders generated by the program.

P.S. this application is really well made,
props to the dev!

**Reply by WhynotFulp** [reply]
Do you know how to download a realm using this mod?
